client/components/site-selector/index.jsx

```diff
@@ -431,15 +447,22 @@ const SiteSelector = React.createClass( {
 			</div>
 		);
 	}
-} );
+}
 
 export default connect( ( state ) => {
 	const user = getCurrentUser( state );
 	const visibleSiteCount = get( user, 'visible_site_count', 0 );
+
 	return {
+		sites: getSites( state ),
 		showRecentSites: get( user, 'visible_site_count', 0 ) > 11,
 		recentSites: getPreference( state, 'recentSites' ),
 		siteCount: get( user, 'site_count', 0 ),
 		visibleSiteCount: visibleSiteCount,
+		getSite: getSite.bind( null, state ),
```

Comment:
ah, you covered this later on in https://github.com/Automattic/wp-calypso/pull/13094/files#r112195983